src/styles/partials/components/_summary-list.scss

```diff
@@ -0,0 +1,67 @@
+.dg_summary-list {
+  color: $gray;
+  background-size: cover;
+  background-color: $gray-lightest;
+}
+  .dg_summary-list-items {
+    display: flex;
+    align-items: flex-start;
+    flex-wrap: wrap;
+  p {
+    margin-bottom: 0px;
+  }
+  .hide {
+    visibility: collapse;
+  }
+  a {
+    text-decoration: none;
+    background-color: transparent;
+    &:hover {
+      text-decoration: underline;
+    }
+  }
+}
+.dg_summary-list-items > * {
+  flex: 1 100%;
+  padding: $padding-small;
+}
+
+.heading {
```

Comment:
these next couple class should be a little more specific (i think)

Reply:
please advice what is your preferred classname ?